### PR TITLE
Calls trial fix

### PR DESCRIPTION
**Pull request by Phrynobatrachus**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -149,7 +149,7 @@ TEMPLATES_DIR=templates
 PLUGIN_PACKAGES ?= mattermost-plugin-antivirus-v0.1.2
 PLUGIN_PACKAGES += mattermost-plugin-autolink-v1.2.2
 PLUGIN_PACKAGES += mattermost-plugin-aws-SNS-v1.2.0
-PLUGIN_PACKAGES += mattermost-plugin-calls-v0.7.1
+PLUGIN_PACKAGES += mattermost-plugin-calls-v0.8.0-test
```
Comment (streamer45) [inline Makefile:152]:
```suggestion
PLUGIN_PACKAGES += mattermost-plugin-calls-v0.8.0-test2
```

**Comment by cpoile** [comment]
Test worked, after @nickmisasi manually completed the cloud trial process. He's looking into what's going wrong, but from our side we're doing what we should to unlock calls for cloud trials.

**Pull request closed without merging**